src/server.js

```diff
@@ -0,0 +1,59 @@
+require('require-self-ref')
+const {checkWatcher, startNPMWatcher} = require('./watchNPM')
+const redis = require('redis')
+const logger = require('~/src/logging').logger(module)
```

Comment:
Your `logger` relies on the config being loaded (logLevel). Since this is the entry point of your app, move your config to the top of the file (below the `require('require-self-ref')` and make sure that the `logger` and any other part of your app sits below it.